tests/Feature/InvoiceFacadeTest.php

```diff
@@ -72,12 +71,8 @@ private function bootstrapGacela(): void
                     ->setDomain('domain.com')
                     ->setReceiver('receiver')
                     ->setSendableRange(1_000, 10_000)
-                    ->addBackend(
-                        'username',
-                        (new LnBitsBackendConfig())
-                            ->setApiEndpoint('http://localhost:5000')
-                            ->setApiKey('XYZ'),
-                    )->jsonSerialize(),
+                    ->addBackendsAsJson(__DIR__ . DIRECTORY_SEPARATOR . 'nostr.json')
```

Comment:
👀 This is the new method where you can specify your custom JSON file with the multiple user settings.